assets/styles/components/NavBar.scss

```diff
@@ -1,161 +1,77 @@
-.NavBarItems{
-    background: linear-gradient(90deg, rgb(0,184,255) 0%, rgba(73,0,255,1) 100%);
```

Comment:
Try following our color palette, using the SCSS variables. See app.scss for reference.